src/python/deepgnn/migrate.py

```diff
@@ -0,0 +1,44 @@
+"""
+Script to migrate to new versions of DeepGNN.
+
+pip install google-pasta
+"""
+import argparse
+from pathlib import Path
+import pasta
+from pasta.augment import rename
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Migrate to new versions of DeepgNN.")
+    parser.add_argument(
+        "--script_dir", type=str, required=True, help="Directory to migrate."
+    )
+    args = parser.parse_args()
+
+    for filename in Path(args.script_dir).glob("**/*.py"):
+        if filename.name.endswith("migrate.py"):
```

Comment:
Why is this needed? We are only migrating external repos, right?

Reply:
I used it on DeepGNN since it is easier. I'll make it a full path so its moer safe.

Reply:
But we do want to migrate users code if it is called migrate.py too, right?

Reply:
I'll change to src/python/deepgnn/migrate.py.